### ISSUE TITLE
"Dependency "mapbox-scenekit" has no shared framework schemes..."

### ISSUE THREAD

**Issue by ghost** [mapbox/mapbox-scenekit#15]
I created a Cartfile with the line `github "mapbox/mapbox-scenekit" "master"` (wouldn't do anything without "master"). When I ran `carthage update --platform iOS`, this is what I got: 

```
$ carthage update --platform iOS
Please update to the latest Carthage version: 0.30.1. You currently are on 0.29.0
*** Fetching mapbox-scenekit
*** Fetching mapbox-events-ios
*** Checking out mapbox-scenekit at "197adf3d90df52d20f58cc14e54745b202000ec1"
*** Checking out mapbox-events-ios at "v0.4.2"
*** xcodebuild output can be found in /var/folders/8t/lxxycwnx6p53z4hzs6gbygqh0000gn/T/carthage-xcodebuild.3IGvuH.log
*** Downloading mapbox-events-ios.framework binary at "v0.4.2"
*** Building scheme "MapboxMobileMetrics" in MapboxMobileEvents.xcodeproj
*** Skipped building mapbox-scenekit due to the error:
Dependency "mapbox-scenekit" has no shared framework schemes for any of the platforms: iOS
```

**Comment by 1ec5** [comment]
This is fixed by #16, right? If so, you can have GitHub automatically close this issue by putting `Fixes #16` or `Closes #16` in the PR description.

**Comment by ghost** [comment]
Oh nice! I didn't know that.
